python/how-to-write-list-to-file.md

```diff
@@ -0,0 +1,15 @@
+# How to write list to file
+
+```python
+lst = ["abc", "def", "ghi"]
+f = open("demofile.txt", "w")
```

Comment:
I believe using the with statement is a better alternative and a best practice in modern python. 
```python
outfile = os.path.join(os.environ['USERPROFILE'], "Documents\\listoutout.txt")
lst = [1,2,3,4,5]
with open(outfile,"w+") as out:
	for i in lst:
		out.write(str(i) + "\n")
```